vnext/ReactWindowsCore/DevSettings.h

```diff
@@ -45,10 +46,16 @@ enum class JSIEngineOverride : int32_t {
   Last = V8Lite
 };
 
+struct ScriptMetadata {
+  uint64_t scriptVersion;
+  std::string bytecodeFilename;
```

Comment:
>bytecodeFilename [](start = 14, length = 16)

let's be consistent in naming, some places it's bytecodeFilename, some it's bytecodePath (eg: ReactInstanceWin32.cpp)

Reply:
bytecodeFilename is used pretty consistently throughout this stack. Going to leave this as is.

---
In reply to: [342756230](https://github.com/microsoft/react-native-windows/pull/3591#discussion_r342756230) [](ancestors = 342756230)